File: geohelp-backend/src/main/java/ch/geohelp/geohelpbackend/data/Emergency.java
package ch.geohelp.geohelpbackend.data;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@AllArgsConstructor
@Getter
@Setter

public class Emergency {
    long eventID;
    long emergencyID;
    boolean approved;
    boolean accepted;
    String responseMessage;
    double longitude;
    double latitude;
}

File: geohelp-organizer-app/src/app/api/emergency.ts
export class Emergency {

  public eventID: number;
  public emergencyID: number;
  public approved: boolean;
  public accepted: boolean;
  public responseMessage: string;
  public longitude: number;
  public latitude: number;


  constructor(eventID: number, emergencyID: number, approved: boolean, accepted: boolean, responseMessage: string, longitude: number, latitude: number) {
    this.eventID = eventID;
    this.emergencyID = emergencyID;
    this.approved = approved;
    this.accepted = accepted;
    this.responseMessage = responseMessage;
    this.longitude = longitude;
    this.latitude = latitude;
  }

}
